modules/ossm-cr-status.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-reference-smcp.adoc
+
+[id="ossm-cr-status_{context}"]
+= status parameter
+
+"The `spec.status` parameter describes the current state of your service mesh. This information is generated by the Operator and is read-only."
```

Comment:
`status` is hierarchically on the same level as `spec`, see here: https://github.com/maistra/api/blob/maistra-2.1/docs/crd/maistra.io_ServiceMeshControlPlane_v2.adoc#servicemeshcontrolplane---maistraiov2

Reply:
Moved the heading level in the TOC to same as `spec`